src/explore-education-statistics-frontend/src/components/Link.tsx

```diff
@@ -11,7 +12,15 @@ export type LinkProps = {
   prefetch?: boolean;
   to?: string | UrlLike;
   unvisited?: boolean;
-} & AnchorHTMLAttributes<HTMLAnchorElement>;
+} & AnchorHTMLAttributes<HTMLAnchorElement> &
+  AnalyticProps;
+
+export interface AnalyticProps {
```

Comment:
Would put this somewhere else in the frontend project e.g. in `googleAnalyticsService` file as this would be a more expected location.